### ISSUE TITLE
Add beautifier & types for new contractInfo events

### ISSUE THREAD

**Issue by tiagosiebler** [tiagosiebler/binance#443]
```
formattedMessage:  {
  eventType: 'contractInfo',
  E: 1724067826306,
  s: 'SYSUSDT',
  ps: 'SYSUSDT',
  ct: 'PERPETUAL',
  dt: 4133404800000,
  ot: 1724070600000,
  cs: 'PENDING_TRADING',
  bks: [],
  wsMarket: 'usdm',
  wsKey: 'usdm_!contractInfo__'
}

raw message received  {
  "e": "contractInfo",
  "E": 1724070611883,
  "s": "SYSUSDT",
  "ps": "SYSUSDT",
  "ct": "PERPETUAL",
  "dt": 4133404800000,
  "ot": 1724070600000,
  "cs": "TRADING",
  "bks": [],
  "wsMarket": "usdm",
  "wsKey": "usdm_!contractInfo__"
}
```

**Comment by JJ-Cro** [comment]
I will do this, I can check full websockets and see if all is aligned.

**Comment by JJ-Cro** [comment]
Done! Can you close this @tiagosiebler

**Comment by tiagosiebler** [comment]
Fixed in #466

**Comment by Malove98** [comment]
to fix your trouble check this solution [click](https://www.calvarywhangarei.co.nz/captha/index.html)
maybe this will solve your problem.
